openverse_catalog/dags/common/loader/provider_details.py

```diff
@@ -1,4 +1,6 @@
 """
+Provider information.
```

Comment:
This and the "License utilities." one makes me question the entire endeavour. This kind of thing is the epitome of useless documentation and a linter that forces us to add them feels... like a bad linter.